src/main/java/customer/Customer.java

```diff
@@ -0,0 +1,47 @@
+package customer;
+
+/**
+ * Represents customers of CliRental
+ */
+public class Customer {
+
+    private String username;
+    private int age;
+    private int contactNumber;
```

Comment:
Perhaps we can store contactNumber as a String??

Reply:
String would be better, or at least Long